chapter09/compose.yaml

```diff
@@ -0,0 +1,288 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This seems to be a copy and needs to be a symbolic ling to the file in the parent directory

Reply:
I don't quite understand what you mean?